src/renderer/menu/MenuBar.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import zIndexes from '../zIndexes'
 import UniverseIcon from '@mui/icons-material/Settings'
-import LightingIcon from '@mui/icons-material/LightBulb'
+import LightingIcon from '@mui/icons-material/Lightbulb'
```

Comment:
A minor issue that surfaced during the build were a few miscapitalized imports, which presumably weren't a problem on macOS and Windows where file systems are usually case-insensitive (as opposed to Linux).